exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -116,11 +115,15 @@ def invalid_url?(url)
           true
         end
 
+        def trace_request
```

Comment:
Small suggestion to broaden expectations for use-cases, and to reduce the cognitive dissonance of `trace_request ... untraced`
```suggestion
        def around_request
```

It would also be really useful to document this method - while it is private, it is expected to be used as an extension point.

Reply:
Updated!